conf/services/personal-account.cy.yml

```
@@ -1,11 +1,62 @@
+  - Ar y dudalen i fwrw golwg dros negeseuon, nid yw’r cysylltiad ‘ewch yn syth i’r prif gynnwys’ yn gweithio ar gyfer y bobl hynny sy’n defnyddio technoleg gynorthwyol.
+  - Ar y dudalen i nodi cyfeiriad rhyngwladol, does dim neges naid sy’n dweud wrth bobl bod y sesiwn ar fin cyrraedd ei therfyn amser. Mae hyn yn ddryslyd i bobl sy’n defnyddio darllenwyr sgrin.
+  - Wrth we-lywio ar frig y dudalen, mae pobl sy’n defnyddio bysellfwrdd i dabio drwy’r cynnwys yn gweld bod y cysylltiad wrth actifadu’r cyfrif yn ehangu’n gwymplen ac mae hyn yn digwydd drwy’r gwasanaeth cyfan. Mae’r gwymplen yn aros ar agor hyd yn oed pan fydd y defnyddiwr yn gwe-lywio i ffwrdd ohoni.  Mae’n rhaid i’r defnyddiwr we-lywio yn ôl ati i’w chau.
+  - Ar y dudalen i nodi cyfeiriad rhyngwladol, mae’r negeseuon gwall yn rhy gyffredinol. Maent hefyd yn cael eu hailadrodd ac nid ydynt yn rhoi disgrifiad da.
+  - Ar y dudalen i nodi cyfeiriad rhyngwladol, nid yw’r opsiwn i gwblhau’r ffurflen yn awtomatig yn ymddangos. Mae’r opsiwn hwn yn helpu defnyddwyr i lenwi meysydd cyfeiriad yn awtomatig sy’n lleihau gwallau.
+  - Ar y dudalen i argraffu crynodeb yswiriant gwladol, mae cyfeiriad e-bost yn ymddangos dros 2 linell pan fydd y porwr wedi ei chwyddo’n sylweddol. Bydd hyn yn effeithio ar ddefnyddwyr â golwg gwan neu’r defnyddwyr hynny sy’n ffafrio darllen y dudalen mewn un golofn.
+  - Ar y dudalen cyfrif treth personol, mae cynnwys Yswiriant Gwladol yn gorgyffwrdd ac yn anodd ei ddarllen pan fydd y dechneg bwlch rhwng testun yn cael ei defnyddio. Mae bwlch rhwng testun yn gosod mwy o le rhwng y cynnwys i helpu’r defnyddiwr i’w ddarllen.
+  - Ar y dudalen i argraffu crynodeb yswiriant gwladol, nid yw’r cyferbynnedd lliw yn bodloni’r gofynion sylfaenol. Mae hyn yn ei gwneud yn anodd gweld a darllen yn enwedig i’r defnyddwyr hynny sydd â golwg gwan neu sy’n lliwddall.
+  - Ar y dudalen i ddewis credydau treth, nid yw’r cyferbynnedd lliw yn bodloni’r gofynion sylfaenol. Mae hyn yn ei gwneud yn anodd gweld a darllen yn enwedig i’r defnyddwyr hynny sydd â golwg gwan neu sy’n lliwddall.
+milestones:
+  - description: |
+      Ar y dudalen i newid y prif gyfeiriad, mae 3 chysylltiad ‘newid’ nad ydynt yn ddisgrifiadol. Nid yw pobl sy’n defnyddio darllenwyr sgrin yn gallu dweud pa gysylltiad sy’n eu helpu i wneud y dasg dan sylw. Nid yw hyn yn bodloni cyfeirnod 1.3.1 (Info and Relationships (Level A)), Canllawiau Hygyrchedd Cynnwys y We.
+    date: 2023-03-31
+  - description: |
+      Ar y dudalen i newid y prif gyfeiriad, mae 3 chysylltiad ‘newid’ nad ydynt yn ddisgrifiadol. Nid yw pobl sy’n defnyddio darllenwyr sgrin yn gallu dweud pa gysylltiad sy’n eu helpu i wneud y dasg dan sylw. Nid yw hyn yn bodloni cyfeirnod(au) Maen Prawf Llwyddiant 2.4.4 (Link Purpose - in context (Level A)) a Maen Prawf Llwyddiant 2.4.9 (Link Purpose - Link Only - (Level AAA)), Canllawiau Hygyrchedd Cynnwys y We.
```

Not like I know any Welsh, but I'd have thought these two lines (28 & 31) should be different to one another, to reference main (14) and postal address (15) accordingly?

```suggestion
      Ar y dudalen i newid y cyfeiriad post, mae 3 chysylltiad ‘newid’ nad ydynt yn ddisgrifiadol. Nid yw pobl sy’n defnyddio darllenwyr sgrin yn gallu dweud pa gysylltiad sy’n eu helpu i wneud y dasg dan sylw. Nid yw hyn yn bodloni cyfeirnod(au) Maen Prawf Llwyddiant 2.4.4 (Link Purpose - in context (Level A)) a Maen Prawf Llwyddiant 2.4.9 (Link Purpose - Link Only - (Level AAA)), Canllawiau Hygyrchedd Cynnwys y We.
```

---

We can't make such changes without discussing with WLU. If you want to get it checked then please add a comment in the ticket

---

Changes made